### DISCUSSION TITLE
modulo cronômetro

### DISCUSSION THREAD

**Discussion by nomegustaa** [balta-io/2801#63] [Perguntas e Respostas] [answered]
Boa tarde. Quando eu tento colocar um condição que compara a variavel type igual a string "m", dá um erro

ERRO: O operador "==" não pode ser aplicado a operandos dos tipos "char" e "string"

Alguém pode me ajudar por favor? 

if (type == "m"){
  int multiplicador = 1;
}

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @nomegustaa, como vai?

Preciso ver seu código mais completo, mas provavelmente sua variável `type` é do tipo `char` e quando você utiliza `""` é uma `string`.
Então comparar `type` com `"m"` é impossível, pois está tentando comparar `char` com `string`.

Você pode alterar o tipo da variável `type` para `string` ou utilizar `'m'` ao invés de `"m"`.
